### PR TITLE
Libsbml src structure

### PR DESCRIPTION
**Pull request by skeating**
## Description

Makes liblx more like sbml so we can use each interchangeably

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated all documentation necessary.
- [ ] I have checked spelling in (new) comments.

## Testing
- [X] Testing is done automatically and codecov shows test coverage
- [ ] This cannot be tested automatically

### CODE REVIEW

File: src/liblx/util/memory.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+ * sbmlStubs.cpp
```
Comment (mattagape) [inline src/liblx/util/memory.cpp:2]: This needs file name etc. changing

File: src/liblx/util/memory.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
+ * sbmlStubs.h
```
Comment (mattagape) [inline src/liblx/util/memory.h:2]: Need to change filename etc

File: src/liblx/util/util.cpp [hunk 3]
```diff
@@ -5,7 +5,8 @@
  *      Author: Timothy Spain
  */
 
-#include "sbmlStubs.h"
+#include "util.h"
```
Comment (mattagape) [inline src/liblx/util/util.cpp:8]: the name of the file in the comment at the top needs changing

File: src/liblx/util/util.h [hunk 4]
```diff
@@ -11,34 +11,22 @@
 #include <ctype.h>
```
Comment (mattagape) [inline src/liblx/util/util.h:11]: You should change filename in top comment and also the names of the `#ifndef` etc.

**Review by OrodRazeghi — APPROVED** [review]

File: CMakeLists.txt [hunk 5]
```diff
@@ -194,56 +194,8 @@ include(CPack)
 option(WITH_EXAMPLES "Compile the libLX example programs."  OFF)
 
 # Which language bindings should be built
-option(WITH_CSHARP   "Generate the C# language interface for libLX."     OFF)
```
Comment (mattagape) [inline CMakeLists.txt:197]: I'm guessing the reason all these language bindings have been deleted is because we haven't implemented them yet?

**Review by mattagape — CHANGES_REQUESTED** [review]

**Pull request merged**